package.json

```
@@ -245,8 +245,8 @@
     "webpack-merge": "5.7.3"
   },
   "engines": {
-    "node": "16.13.0",
-    "npm": "8.1.0"
+    "node": ">=16.13.0",
+    "npm": ">=8.1.0"
```

```suggestion
    "node": "16",
    "npm": "8"
```

I wonder if this is better, so it's even less restrictive, in theory only the major version should matter.

---

Done.  Thanks.